R/categories.R

```diff
@@ -239,3 +239,70 @@ setMethod("lapply", "Categories", function (X, FUN, ...) {
     X@.Data <- lapply(X@.Data, FUN, ...)
     return(X)
 })
+
+#' Change the id of a category for a categorical variable
+#' 
+#' @param variable the variable in a crunch dataset that will be changed
+#' @param from the (old) id identifying the category you want to change
+#' @param to the (new) id for the category
+#' @return \code{variable} with an the new category id
+#' @examples
+#' \dontrun{
+#' ds$country <- changeCategoryID(ds$country, 2, 6)
+#' ds$petloc <- changeCategoryID(ds$petloc, 2, 6)
```

Comment:
I intended for them to be an example of a categorical and a categorical array. although, like you you said, without a description of the dataset it's not very clear. I can drop the second if we want.